pyzeebe/worker/worker.py

```diff
@@ -31,27 +31,28 @@ def __init__(self, name: str = None, request_timeout: int = 0, hostname: str = N
         self.name = name or socket.gethostname()
         self.request_timeout = request_timeout
         self.tasks = []
+        self._task_threads: List[Thread] = []
 
-    def work(self):
-        with ThreadPoolExecutor(thread_name_prefix='zeebe-task') as executor:
-            for task in self.tasks:
-                executor.submit(self._handle_task, task)
+    def work(self, stop_event: Event = None):
+        for task in self.tasks:
+            task_thread = Thread(target=self._handle_task, args=(task, stop_event), daemon=True)
```

Comment:
Check if `daemon=True` should be used here

Reply:
Daemon should be the default of the current thread. Meaning if the calling (running) thread is a daemon so will the task_thread.